tests/integration/targets/instance_type_change/tasks/main.yaml

```diff
@@ -32,11 +35,12 @@
         state: present
       register: resize_disks
 
-    - assert:
+    - name: Assert instance is resized with resizing disks
+      assert:
         that:
           - resize_disks.changed
           - resize_disks.instance.type == "g6-standard-1"
-          - resize_disks.disks[0].size == type_info.types[0].disk - 512
+          - resize_disks.instance.specs.disk == type_info.types[0].disk
```

Comment:
This should be good for now, although in the future we may want to implement logic in the instance module to wait for all disks to enter `ready` status after type change events finish 👀